src/classes/CW_AudienceGroupController.cls

```diff
@@ -0,0 +1,543 @@
+/**
+* File Name   :    CW_AudienceGroupController.cls
+* Description :    B-110228/B-110230/B-109518
+*                  CIM, BI, Marketer must be able to View,Edit,Delete audience groups 
+*				   that are linked to the campaign 
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 01/04/2014 Cristina P.	Intial
+* 0.2 08/04/2014 Cristina P.	B-109513 - download the audience group template
+* 0.3 24/04/2014 Alex Panka   D-106803, B-110234
+*
+*/
+
+public with sharing class CW_AudienceGroupController {
```

Comment:
* This class seems to have grown very large and is becoming difficult to maintain. I suggest we consider refactoring to reduce the number of methods.
* Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.